OpenEdXMobile/res/menu/find_courses.xml

```diff
@@ -5,6 +5,7 @@
     <item android:id="@+id/menu_item_search"
           android:title="@string/search_for_courses"
           app:actionViewClass="android.support.v7.widget.SearchView"
+          android:icon="@drawable/abc_ic_search_api_material"
```

Comment:
Using resources declared `private` within a library isn't a best practice. A better approach would be to copy this resource to our project avoiding any future conflicts.